### PR TITLE
AV-191987: Added ako-gateway-api image entry in SRP data

### PR DESCRIPTION
**Pull request by jeyapradeen-avi**

### CODE REVIEW

**Comment by akshayhavile** [comment]
Can one of the admins verify this patch?

File: hack/jenkins/save_build.sh [hunk 1]
```diff
@@ -92,22 +91,31 @@ sudo /srp-tools/srp provenance action stop --working-dir $WORKSPACE/provenance
...
 IMAGE_DIGEST=`sudo docker images $CI_REGISTRY_IMAGE_AKO  --digests | grep sha256 | xargs | cut -d " " -f3`
 echo $IMAGE_DIGEST
-sudo /srp-tools/srp provenance add-output package.oci --set-key=ako-image --action-key=ako-bld --name=${CI_REGISTRY_IMAGE_AKO}  --digest=${IMAGE_DIGEST} --working-dir $WORKSPACE/provenance
+sudo /srp-tools/srp provenance add-output package.oci --set-key=ako-image --action-key=ako-build --name=${CI_REGISTRY_IMAGE_AKO}  --digest=${IMAGE_DIGEST} --working-dir $WORKSPACE/provenance
 
+CI_REGISTRY_IMAGE_AKO_OPERATOR=$CI_REGISTRY_PATH/ako-operator
 IMAGE_DIGEST=`sudo docker images $CI_REGISTRY_IMAGE_AKO_OPERATOR  --digests | grep sha256 | xargs | cut -d " " -f3`
 echo $IMAGE_DIGEST
-sudo /srp-tools/srp provenance add-output package.oci --set-key=ako-operator-image --action-key=ako-bld --name=${CI_REGISTRY_IMAGE_AKO_OPERATOR}  --digest=${IMAGE_DIGEST} --working-dir $WORKSPACE/provenance
+sudo /srp-tools/srp provenance add-output package.oci --set-key=ako-operator-image --action-key=ako-build --name=${CI_REGISTRY_IMAGE_AKO_OPERATOR}  --digest=${IMAGE_DIGEST} --working-dir $WORKSPACE/provenance
```
Comment (akshayhavile) [inline hack/jenkins/save_build.sh:102]: Can we please take out ako-operator related changes from SRP? Reason: We do not want to generate ako-operator image as part of helm processing. So we will skip ako-operator step in jenkins job also.  So generating provenance data also will not be useful in that case.

**Review by akshayhavile — APPROVED** [review]
We will work on removing ako-operator code later.

**Comment by akshayhavile** [comment]
build ako

File: hack/jenkins/save_build.sh (hunk 1 above)
Comment (jeyapradeen-avi) [reply hack/jenkins/save_build.sh:102]: As per the discussion will do change later

**Review by DixitAakash — APPROVED** [review]

**Pull request merged**